Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.70.0-bullseye AS chef
```

Comment:
Current Rust version is 1.71.1, is this ok?

Reply:
oh, i'll update